test/e2e-helm/e2e_helm_metrics_test.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_helm_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
+	kbtestutils "sigs.k8s.io/kubebuilder/test/e2e/utils"
+)
+
+var _ = Describe("Exporting metrics from a Helm Project", func() {
+
+	const curlImage = "curlimages/curl:latest"
+	var controllerPodName string
+
+	Context("built with operator-sdk", func() {
+		BeforeEach(func() {
+			By("Enabling Prometheus via the kustomization.yaml")
+			Expect(kbtestutils.UncommentCode(
+				filepath.Join(tc.Dir, "config", "default", "kustomization.yaml"),
+				"#- ../prometheus", "#")).To(Succeed())
+
+			By("Checking the kustomize setup")
+			err := tc.Make("kustomize")
+			Expect(err).Should(Succeed())
+
+			By("Building the project image")
+			err = tc.Make("docker-build", "IMG="+tc.ImageName)
+			Expect(err).Should(Succeed())
+
+			if IsRunningOnKind() {
+				By("Loading the project image into Kind cluster")
+				err = tc.LoadImageToKindCluster()
+				Expect(err).Should(Succeed())
+			}
+
+			By("Deploying project on the cluster")
+			err = tc.Make("deploy", "IMG="+tc.ImageName)
+			Expect(err).Should(Succeed())
```

Comment:
All of this should be part of the test itself, not setup, since these are testing operator project deployment.

Reply:
done